### DISCUSSION TITLE
Multi: Beschriftung wahr fehlt

### DISCUSSION THREAD

**Discussion by ppb1308** [inventwo/ioBroker.vis-inventwo#331] [Q&A] [answered]
Hallo,
im Multi Widget fehlt bei mir trotz Neu-Installation u.a. "Beschriftung wahr".
Lt. Wiki sollte es unter Zustand[0] als drittes Feld verfügbar sein, dort ist bei mir aber "View[1]".
Was kann ich tun, um eine Beschriftung zu pflegen, die bei dem angegebenen Wert des Objektes ausgegeben werden soll?
Gruß,
Peter
![Multi_Beschriftung_wahr_fehlt](https://user-images.githubusercontent.com/16593386/141980299-7b43f263-8b63-4f7c-83fc-b5a8322d878a.png)

**Comment by sigi2345** [comment, marked as answer]
> Hallo, im Multi Widget fehlt bei mir trotz Neu-Installation u.a. "Beschriftung wahr". Lt. Wiki sollte es unter Zustand[0] als drittes Feld verfügbar sein, dort ist bei mir aber "View[1]". Was kann ich tun, um eine Beschriftung zu pflegen, die bei dem angegebenen Wert des Objektes ausgegeben werden soll? Gruß, Peter 
----------------------------------------------------------
Klicke mal auf Zustand [0]   Icon Pfeil nach unten und dann noch mal

**Reply by ppb1308** [reply]
Danke, das hab ich nicht gewusst.
